internal/github/checkPendingCI.go

```diff
@@ -119,9 +119,9 @@ func checkTeamMemberActivity(ctx context.Context, client *github.Client, owner,
 	}
 
 	for _, comment := range comments {
-		log.Printf("Checking comment by %s at %s for PR #%d of repo %s", comment.User.GetLogin(), comment.CreatedAt.Format(time.RFC3339), prNumber, repo)
+		log.Logger.Info("Checking comment by team member", "user", comment.User.GetLogin(), "created_at", comment.CreatedAt.Format(time.RFC3339), "PR", prNumber, "repository", repo)
```

Comment:
In another file, you use `PR Number` in the key-values. Should be consistent with either one (I prefer single word keys but no strong opinion on which you choose.)

Same comment for repository vs repo vs Repo

Reply:
Changed terms to be more consistent